native/python/bindings/setup.py

```diff
@@ -17,6 +17,9 @@
                            sources=["zusf_py_wrap.cxx", "zusf_py.cpp", f"{C_PATH}/zusf.cpp", f"{C_PATH}/zut.cpp"],
                            language="c++",
                            include_dirs=[chdsect],
+                           extra_objects=[
+                               f"{swig_build_path}/zshmem.o",
+                           ],
```

Comment:
This technically can be removed after the PR below is merged:
- https://github.com/zowe/zowe-native-proto/pull/518